client/blocks/plan-storage/bar.jsx

```diff
@@ -58,7 +58,7 @@ class PlanStorageBar extends Component {
 		return (
 			<div className={ classes }>
 				<ProgressBar
-					className="plan-storage__bar"
+					className="plan-storage__storage-bar"
```

Comment:
🤕 you're right. I took the code and the className is duplicated. :-( 
I'm sorry folks.